app/src/main/res/values/strings.xml

```diff
@@ -1004,4 +1004,8 @@
     <!--Action button for setting Travel Discovery as default search engine-->
     <string name="travel_dialog_2_action">SET AS DEFAULT</string>
 
+    <!--Download Notification-->
+    <string name="download_notification_title">Firefox Lite is going to download this file</string>
+    <string name="download_notification_message">Do you want to download from this page?</string>
```

Comment:
Please add comments so the l10n team will know when these two strings will be used, and where.

Reply:
"from this page" seems wired to me. Maybe the copy/UX team will have a better wording here.